packages/paste-core/components/timeline/src/types.ts

```diff
@@ -0,0 +1,99 @@
+import type { BoxProps } from "@twilio-paste/box";
+import type { IconWrapperProps } from "@twilio-paste/icons/esm/helpers/IconWrapper";
+import type { GenericIconProps } from "@twilio-paste/icons/esm/types";
+import type { HTMLPasteProps } from "@twilio-paste/types";
+
+export type Orientation = "horizontal" | "vertical";
+
+export type TimelineProps = {
+  /**
+   * The orientation of the timeline
+   *
+   * @type {"horizontal" | "vertical"}
+   * @default "vertical"
+   * @memberof TimelineProps
+   */
+  // orientation?: "horizontal" | "vertical";
```

Comment:
Just a comment for other reviewers, I'm fine with leaving this in if we are implementing at some point in the future

Reply:
hmm I'd rather remove all the commented stuff and keep it in a branch for the future